client/locations-list-map.js

```diff
@@ -1527,6 +1530,7 @@ function getDatasetFilename(dataset, countryCode) {
   return `/${dataset}-${countryCode}.json`;
 }
 
+// Grab the country codes of other countries and load their json data too
```

Comment:
Period. :)

Also, it's preferable to explain the side effects of a function.  (Honestly, it's preferable to not even have global side effects, but alas...too big of a refactor).